Jenkinsfile_CNP

```diff
@@ -76,7 +76,9 @@ env.PACT_BROKER_FULL_URL = "https://pact-broker.platform.hmcts.net"
 env.AMQP_HOST = "rd-servicebus-aat.servicebus.windows.net"
 env.CRD_TOPIC_NAME = "rd-caseworker-topic-aat"
 env.JRD_TOPIC_NAME = "rd-judicial-topic-aat"
-env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
```

Comment:
Note: revert back to AAT RAS instance prior to master merge

Reply:
Resolved in 929cbbdbda46f5be83a9db9d329ba1faa43eedbe.